### DISCUSSION TITLE
Detailed place description for desktop app

### DISCUSSION THREAD

**Discussion by SnowCat007** [organicmaps/organicmaps#6595] [Ideas]
Having a quick description for a place you're looking at is something I use quite a lot, but this feature isn't available on the desktop version of the app. Instead it points you to online articles from Wikipedia, which is fine, but I'm not always connected to the internet when using the app, and as such I sometimes find myself needing to know general information about a place I'm travailing to while offline. The mobile version of this app does this perfectly with what looks like full versions of the Wikipedia articles which can be viewed even when I move the files from one device to another without being online. From what I've seen the articles are stored in the map files you download, so I'm wondering why the desktop app doesn't also display the offline information that's stored in the files it has? Can this feature be added to the desktop app?
Thanks for all the work the members of the team have put into the mobile and desktop apps, I greatly love them, and use them all the time.

**Comment by biodranik** [comment]
How would you conveniently display (potentially long) articles in the dialog? See also #6598
